test/integration-tests/extension.test.ts

```diff
@@ -29,9 +29,9 @@ function getRootWorkspaceFolder(): vscode.WorkspaceFolder {
 
 export const globalWorkspaceContextPromise: Promise<WorkspaceContext> = (async () => {
     const workspaceFolder = getRootWorkspaceFolder();
-    const ext = vscode.extensions.getExtension<Api>("sswg.swift-lang");
+    const ext = vscode.extensions.getExtension<Api>("swiftlang.swift-lang");
```

Comment:
I find this to be inconsistent with other places where you refactored them as `"swiftlang.vscode-swift"`. Same below.

Reply:
Changed, thanks